integration-tests/testconfig/configs_embed.go

```diff
@@ -0,0 +1,21 @@
+//go:build embed
+// +build embed
+
+package testconfig
+
+import "embed"
+
+//go:embed default.toml
+//go:embed automation/automation.toml
+//go:embed functions/functions.toml
+//go:embed keeper/keeper.toml
+//go:embed log_poller/log_poller.toml
+//go:embed node/node.toml
+//go:embed ocr/ocr.toml
+//go:embed vrfv2/vrfv2.toml
+//go:embed vrfv2plus/vrfv2plus.toml
+var embeddedConfigsFs embed.FS
+
+func init() {
+	areConfigsEmbedded = true
+}
```

Comment:
you don't need that embedded and noembed configs if you are not planning to compile the tests to a binary like we do in core in some workflows.

if you do need it, though, you should remove all embeds but these two:
```
//go:embed default.toml
//go:embed ocr2/ocr2.toml
```

Because other files do not exist and that will make compilation fail.